microsoft-edge/webview2/releasenotes.md

```diff
@@ -20,6 +20,46 @@ The WebView2 team is delivering updates to the [WebView2 SDK][NuGetGallery] on a
 > [!IMPORTANT]
 > While WebView2 is a preview, the .NET APIs are in the `pre-release package`.  
 
+
+## 0.9.627-prerelease  
+Release Date: 09/08/2020
+
+[NuGet package][NuGetGallery0.9.579] \| minimum Microsoft Edge version 86.0.579.0.  
+
+   > [!IMPORTANT]
+    > **Announcement**: This prerelease will continue to release out of Edge 87 Branch. In the future, pre-release SDK releases will match the Edge Canary builds, whereas our normal releases will be in sync with Edge Stable.
+
+#### General  
+* 
+#### .NET  
+
+*   .NET Binaries are now [Strongly Named]().  \([\#181][GithubMicrosoftedgeWebviewfeedbackIssue181]\).
+
+
+## 0.9.622  
+Release Date: 09/08/2020
```

Comment:
Release date would most likely be next Wednesday. Adding a comment here to ensure we don't forget to update it :)